lib/time/duration.ex

```diff
@@ -424,12 +545,25 @@ defmodule Timex.Duration do
   end
 
   @doc """
-  Time interval since Epoch.
+  Returns the amount of time since Epoch.
+
+  The argument is an atom indicating the type of time units to return.
+
+  The allowed unit type atoms are:
+  - :microseconds
+  - :milliseconds
+  - :seconds
+  - :minutes
+  - :hours
+  - :days
+  - :weeks
+
+  ## Examples
 
-  The argument is an atom indicating the type of time units to return (see
-  convert/2 for supported values).
+  iex> Timex.Duration.now(:seconds)
+  :os.system_time(:seconds)
```

Comment:
So this is a terrible test is it stands as it's purely tautological. That said, it at least demonstrates the behavior for documentation purposes, so I left it in.